src/main/java/com/exasol/dbbuilder/ConnectionDefinition.java

```diff
@@ -0,0 +1,108 @@
+package com.exasol.dbbuilder;
+
+import com.exasol.dbbuilder.objectwriter.DatabaseObjectWriter;
+
+/**
+ * Definition of a connection to another database or service.
+ * <p>
+ * Other terms that are used for this are "named connection" or "connection object". A connection definition is a named
+ * database object that contains a reference to the remote database or service and optionally credentials.
+ * </p>
+ */
+public class ConnectionDefinition extends AbstractDatabaseObject {
+    private final String to;
```

Comment:
Maybe change `to` to `definition`, `destinationConnection` or `targetConnection`?

Reply:
Changed to `target`. Renamed methods accordingly.